src/components/BestItem.jsx

```diff
@@ -0,0 +1,29 @@
+import { 
+    constainer,
+    title,
+    itemGrid,
+} from "./BestItemStyle";
+import ItemList from "./ItemList";
+
+/** @jsxImportSource @emotion/react */
+
+const BestItem = ({ items }) => {
+    const bestItems = items.slice(0, 4);
+
+    return(
+        <div css={constainer}>
+            <h2 css={title}>
+                베스트 상품
+            </h2>
+            
+            <div css={itemGrid} >
+                {bestItems.map((item) => {
+                    return <ItemList key={item.id} item={item}/>
+                })}                
```

Comment:
💊 제안
순회를 돌며 단순히 jsx를 반환할 뿐이니 아래처럼 하시는게 가독성 측면에서 더 좋을 것 같아요.
```suggestion
                {bestItems.map((item) => <ItemList key={item.id} item={item}/> )}                
```